dangerfile.js

```diff
@@ -5,8 +5,20 @@ const { danger, markdown } = require('danger');
 const { exec } = require('child_process');
 const { loadComparison } = require('./scripts/sizeSnapshot');
 
-const parsedSizeChangeThreshold = 300;
-const gzipSizeChangeThreshold = 100;
+const PARSED_SIZE_CHANGE_THRESHOLD = 300;
+const GZIP_SIZE_CHANGE_THRESHOLD = 100;
+const UPSTREAM_REMOTE = 'danger-upstream';
+/**
+ * mapping from bundle id to a human label
+ */
+const LABELS = {
```

Comment:
What do you think of collocating the id and the label data in the same source array?

Reply:
I can work on it if you agree with the strategy.